packages/bff/package.json

```
@@ -37,6 +37,7 @@
     "axios": "1.7.2",
     "dotenv-flow": "4.1.0",
     "express": "4.19.2",
+    "fast-xml-parser": "^4.4.0",
```

With the setup we have in `.npmrc` the package should be by default installed as pinned 🤔 
Anyways, let's remove the `^`

```suggestion
    "fast-xml-parser": "4.4.0",
```